### ISSUE TITLE
Android Auto: custom map settings are only applied when app is started on phone

### ISSUE THREAD

**Issue by martymcbro** [osmandapp/OsmAnd#14171]
### Description
When using OsmAnd+ Android Auto integration with a custom map configurarion (i.e. activated Microsoft-Earth underlay map, activated custom Google-Traffic overlay map, changed road colors), the map configurarion ist not applied when starting up Osmand+ in Android Auto. Instead the standard map configuration without customizations is used. As soon as I open the OsmAnd+ app on the phone, the custom map configuration is activated on the car screen. For the custom configuration the map type "car" was used. The custom map configuration should be displayed on the car screen without the need to open the app on the phone.

### How to reproduce?
- Customize configuration of map type "car" with a Microsoft-Earth underlay map.
- Close the OsmAnd+ app.
- Connect to Android Auto and start OsmAnd+ app on car screen -> standard map configuration is displayed
- Open OsmAnd+ app on phone -> custom map configuration is displayed

### Your Environment
OsmAnd Version: OsmAnd+ 4.1.11
Android/iOS version: Android 12
Device model: Samsung Galaxy S20FE (Snapdragon)

**Maps used (online or offline):**   
Offline map "Deutschland Rheinland-Pfalz"

**Comment by vshcherb** [comment]
It looks like by default another Profile is used (Browse Map or Car). Can you please confirm which Profile is used when you start:
- Open OsmAnd+ app on phone 
- Connect to Android Auto and start OsmAnd+ app on car screen 

As a workaround you can customize Browse Map profile as well probably. If I understand correctly then when you start navigation OsmAnd should automatically switch to Car profile

**Comment by martymcbro** [comment]
- When I start the OsmAnd+ app on the phone (forced closed it before), always the "Car" profile is loaded first.
- When I start the OsmAnd+ Android auto app, while the app on the phone is open, the profile which is displayed in the phone app is also displayed on the car screen. Overlay map and underlay map are loaded correctly. The displayed map on the car screen switches correctly when I switch the profile on the phone.
- When I start the OsmAnd+ Android auto app, while the app on the phone is closed, the map of the "Car" profile is loaded first. I recognize it because of the special POI settings (gas stations) of the car profile which are displayed. However the overlay map and underlay map are not displayed. As soon as I open the app on the phone, the overlay and underlay map is loaded. This also seems to keep working even if I switch to Google maps on the car screen, force close Osmand+ on the phone and start the app on the car screen again. Overlay and underlay map are still displayed. It stops working again on the next day, when I had disconnected from the car for a longer time.
- I configured the underlay map for both the car and the browse profile. So it should be loaded in both of these profiles.

I hope my explanations make some sense and give you a clue what could possibly go wrong.
